pandas/tests/series/test_constructors.py

```diff
@@ -164,22 +183,24 @@ def test_constructor_list_like(self):
 
     @pytest.mark.parametrize('input_vals', [
         ([1, 2]),
-        ([1.0, 2.0, np.nan]),
```

Comment:
I had to remove this case, as it isn't correct. We aren't consistent between `Series(... dtype=str)` and `.astype(str)`, which is maybe unfortunate...

```python
In [4]: pd.Series([1, 2, None], dtype='str').tolist()
Out[4]: ['1', '2', None]

In [5]: pd.Series([1, 2, None], dtype='str').astype('str').tolist()
Out[5]: ['1', '2', 'None']
```